.travis.yml

```diff
@@ -11,6 +14,15 @@ node_js:
 services:
   - redis-server
 
+before_install:
+  # Use a specific version of yarn in CI. This ensures yarn.lock format doesn't change.
+  - curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 1.19.1
```

Comment:
This is so that, for example, the build doesn't fail once yarn v2 is released or when yarn 1.20.0 changes the yarn.lock file format ever so slightly. This does happen from time to time with yarn.